src/main/resources/application.yml

```
@@ -0,0 +1,56 @@
+
+  cloud:
+    aws:
+      region:
+        static: ap-northeast-2
+      s3:
+        bucket: ${bucket-name}
+
+logging:
+  level:
+    org.hibernate.SQL: debug
+    org.hibernate.orm.jdbc.bind: trace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)